alerter.js

```diff
@@ -24,3 +24,22 @@ alertInCelcius(400.5);
 alertInCelcius(303.6);
 console.log(`${alertFailureCount} alerts failed.`);
 console.log('All is well (maybe!)');
+
+//Testing Alerter
+let threshold = 100;
+let actualAlertFailureCount = 0;
+
+function testAlerter(farenheit) {
+    const celcius = (farenheit - 32) * 5 / 9;
+    if(celcius <= threshold){
+        return 200;
+    }
+    else {
+        actualAlertFailureCount += 1;
+        return 500;
+    }
+
+}
+
+expect(networkAlertStub(250)).equals(testAlerter(250));
+expect(alertFailureCount).equals(actualAlertFailureCount);
```

Comment:
This is a weak test case. The test cases should cover all possible input values, so that every expectation on the outcome of production code is explicitly defined.